### PR TITLE
add release to charmhub

### PR DESCRIPTION
**Pull request by paulomach**
## Issue

Auto publish to charmhub. Temporarily using image from [this dockerhub registry](https://hub.docker.com/r/dataplatformoci/pgbouncer)

## Release Notes

- `.github/workflows/ci.yaml` - removed OCI build steps
- `.github/workflows/release.yaml` - release workflow
- `CONTRIBUTING.md` - update instructions
- `metadata.yaml` - updated upstream source

### CODE REVIEW

**Review by marcoppenheimer — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Review by WRFitch — CHANGES_REQUESTED** [review]
LGTM, thanks again for the changes! 

Unfortunately I have to block this PR for the same reasons as the other one in `pgbouncer-operator` - I don't feel this charm is ready for release, so I can't approve automated releases. The two tasks left to complete are integration testing and the db relations.

**Review by WRFitch — APPROVED** [review]
LGTM :)

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -66,21 +38,12 @@ jobs:
...
-      - name: Download image built in build oci image step
-        uses: actions/download-artifact@v2
-        with:
-          name: image
-          path: /tmp
-      - name: Load Image into microk8s
-        run: |
-          /usr/bin/sg microk8s -c "microk8s ctr image import /tmp/image.tar"
       - name: Run integration tests
         # set a predictable model name so it can be consumed by charm-logdump-action
```
Comment (MiaAltieri) [inline .github/workflows/ci.yaml:42]: might want to also remove this comment

**Review by MiaAltieri — APPROVED** [review]
LGTM one comment though :)

**Review by MiaAltieri — APPROVED** [review]
nice 🤩

**Review by marceloneppel — APPROVED** [review]
LGTM!

**Pull request merged**